### ISSUE TITLE
Allow selective install modifications

### ISSUE THREAD

**Issue by LuisMayo** [Heroic-Games-Launcher/HeroicGamesLauncher#2694]
### Feature description

Right now, games with selective downloads allow the user to select what they want to download when installing the game. But we have no way to edit an existing installation to add or remove components. An option to modify an existing game installation would be good

### Alternatives

Re-installing the game, which is not optimal for obvious reasons

### Additional information

Not sure if this has been proposed but I haven't been able to find it. Not sure if it's possible even.

Thanks for your work!

**Comment by arielj** [comment]
@LuisMayo can you confirm if this got fixed with the new DLC and SDL manager in heroic 2.8.0?

**Comment by LuisMayo** [comment]
I can't locate the SDL manager. The DLC manager shows nothing for fortnite for instance, a game which famously includes SDLs

![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/20229636/0f0436de-8416-4531-aefd-87883a51272e)

**Comment by LuisMayo** [comment]
Then maybe this asks for an upstream issue to the Legendary Repo to support SDL modifications without re-installing

**Comment by flavioislima** [comment]
> Then maybe this asks for an upstream issue to the Legendary Repo to support SDL modifications without re-installing

Yes, if Legendary adds support for it the way it handles the DLCs than that would make it easier for Heroic as well.

**Comment by LuisMayo** [comment]
I've now opened an issue in Legendaries' repo: https://github.com/derrod/legendary/issues/569

**Comment by CReimer** [comment]
I stumbled upon the same problem with Fortnite. It looks like upstream implemented the necessary feature.

Any update for Heroic on that front?

**Comment by RuiNtD** [comment]
What's the status of this? https://github.com/derrod/legendary/issues/569 was marked as completed back in June.

**Comment by arielj** [comment]
I don't think anyone is working on this yet

**Comment by gr4veyxrd** [comment]
Bumping this issue. It'd be nice to at least know why such an integral feature has gotten no contributor feedback for over an entire year now.

**Comment by knock2one** [comment]
bumpin that (this would be awesome to see!!)

**Comment by KibSquib48** [comment]
Almost a year later, any progress on this yet?

**Comment by BlackBaroness** [comment]
Any updates?
